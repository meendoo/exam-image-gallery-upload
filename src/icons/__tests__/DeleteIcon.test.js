import React from 'react'
import DeleteIcon from '../DeleteIcon'
import renderer from 'react-test-renderer'

describe('DeleteIcon', () => {
    it('should render as it is', ()=> {
        const spinner = renderer.create(<DeleteIcon />);
        expect(spinner).toMatchSnapshot();
    })
})
