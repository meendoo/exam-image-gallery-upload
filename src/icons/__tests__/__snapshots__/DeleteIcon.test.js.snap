// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteIcon should render as it is 1`] = `
<svg
  className=""
  fill="#000000"
  height="40px"
  viewBox="0 0 70 70"
  width="40px"
  x="0px"
  y="0px"
>
  <g>
    <path
      d="M47.3723145,60H22.6281738c-1.2568359,0-2.3232422-0.9814453-2.4277344-2.2338867l-3.0844727-37.0151367c-0.0229492-0.2788086,0.0712891-0.5541992,0.2607422-0.7602539c0.1894531-0.2055664,0.4560547-0.3227539,0.7358398-0.3227539h33.7753906c0.2797852,0,0.5463867,0.1171875,0.7358398,0.3227539c0.1894531,0.2060547,0.2836914,0.4814453,0.2607422,0.7602539l-3.0849609,37.0151367C49.6950684,59.0185547,48.6291504,60,47.3723145,60z M19.1994629,21.6679688l2.9941406,35.9321289C22.2121582,57.8242188,22.4030762,58,22.6281738,58h24.7441406c0.2285156,0,0.4150391-0.171875,0.434082-0.3999023l2.9946289-35.9321289H19.1994629z"
    />
    <path
      d="M42.1057129,53.5039063c-0.0136719,0-0.0273438-0.0004883-0.0410156-0.0009766c-0.5517578-0.0219727-0.980957-0.487793-0.9584961-1.0395508l0.9648438-23.8291016c0.0219727-0.5517578,0.4931641-0.9853516,1.0395508-0.9584961c0.5517578,0.0219727,0.980957,0.487793,0.9584961,1.0395508L43.104248,52.5444336C43.0827637,53.0825195,42.6394043,53.5039063,42.1057129,53.5039063z"
    />
    <path
      d="M35.0002441,53.5039063c-0.5522461,0-1-0.4477539-1-1V28.6748047c0-0.5522461,0.4477539-1,1-1s1,0.4477539,1,1v23.8291016C36.0002441,53.0561523,35.5524902,53.5039063,35.0002441,53.5039063z"
    />
    <path
      d="M27.8942871,53.5039063c-0.5336914,0-0.9770508-0.4213867-0.9985352-0.9594727L25.9309082,28.715332c-0.0224609-0.5517578,0.4067383-1.0175781,0.9584961-1.0395508c0.5400391-0.0307617,1.0170898,0.40625,1.0395508,0.9584961l0.9648438,23.8291016c0.0224609,0.5517578-0.4067383,1.0175781-0.9584961,1.0395508C27.9216309,53.503418,27.907959,53.5039063,27.8942871,53.5039063z"
    />
    <path
      d="M53.6633301,21.6679688H16.3366699c-0.5522461,0-1-0.4477539-1-1v-1.625c0-1.8808594,1.5297852-3.4106445,3.4106445-3.4106445h32.5058594c1.8803711,0,3.4101563,1.5297852,3.4101563,3.4106445v1.625C54.6633301,21.2202148,54.2155762,21.6679688,53.6633301,21.6679688z M17.3366699,19.6679688h35.3266602v-0.625c0-0.777832-0.6328125-1.4106445-1.4101563-1.4106445H18.7473145c-0.777832,0-1.4106445,0.6328125-1.4106445,1.4106445V19.6679688z"
    />
    <path
      d="M40.5388184,17.6323242H29.4616699c-0.5522461,0-1-0.4477539-1-1v-3.5908203C28.4616699,11.3642578,29.8259277,10,31.5031738,10h6.9936523c1.6772461,0,3.0419922,1.3642578,3.0419922,3.0415039v3.5908203C41.5388184,17.1845703,41.0910645,17.6323242,40.5388184,17.6323242z M30.4616699,15.6323242h9.0771484v-2.5908203C39.5388184,12.4672852,39.0715332,12,38.4968262,12h-6.9936523c-0.5742188,0-1.0415039,0.4672852-1.0415039,1.0415039V15.6323242z"
    />
  </g>
</svg>
`;
